src/base/utils/mac_utils.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2015  Vladimir Golovnev <glassez@yandex.ru>
```

Comment:
Oh, there is gui code in here. You should move "src/base/utils/mac_utils.\*" to "src/gui/mac_utils.\*"
or "src/gui/utils/mac_utils.\*" as you see fit.